### PR TITLE
Fix race drought and other maps

### PR DESCRIPTION
**Pull request by CoWinkKeyDinkInc**
- fixed race drought teams by adding team IDs
- fixed spacing and team colours in creep slap by using the correct colour names, (ie gold=orange)
- fixed the mob spawning in creep slap
- fixed spacing in Monochrome, removed mob entities and pruned the map
- fixed various regions, portals, filters and destroyables in Table Block. Ask someone about the monument modes because all of the modes are set to go through with like 2 minutes into the game.

### CODE REVIEW

File: maps/race_drought/race.xml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-    <team color="yellow" max="1" max-overfill="1">Yellow Car</team>
-    <team color="dark purple" max="1" max-overfill="1">Purple Car</team>
-    <team color="light purple" max="1" max-overfill="1">Pink Car</team>
-    <team color="gold" max="1" max-overfill="1">Orange Car</team>
-    <team color="green" max="1" max-overfill="1">Lime Car</team>
-    <team color="dark aqua" max="1" max-overfill="1">Cyan Car</team>
-    <team color="gray" max="1" max-overfill="1">Gray Car</team>
-    <team color="dark blue" max="1" max-overfill="1">Navy Car</team>
-    <team color="black" max="1" max-overfill="1">Black Car</team>
+    <team id="blue" color="blue" max="1" max-overfill="1">Blue Car</team>
```
Comment (CrazyisCreeps) [inline maps/race_drought/race.xml:4]: Not sure if ids are needed, the map proto doesn't require them

Comment (CoWinkKeyDinkInc) [reply maps/race_drought/race.xml:4]: Are you sure? IDs don't match colour.

Comment (CoWinkKeyDinkInc) [reply maps/race_drought/race.xml:4]: `pgm-maps/maps/race_drought/map.xml - 'team' attribute of 'spawn' element @ line 39: Missing team with ID '--team-black'`

I get these errors from the console, the XML fixed it so yeah lol

**Comment by CrazyisCreeps** [comment]
Maps were moved to overcast repo

**Pull request closed without merging**